### PR TITLE
feat(lambda-tiler): smoke test during deployment

### PR DESCRIPTION
**Pull request by Wentao-Kuang**
When convenient, please have a look the changes and feedback.

### CODE REVIEW

File: packages/lambda-tiler/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "@cogeotiff/core": "^2.2.0",
     "@cogeotiff/source-aws": "^2.2.0",
     "@linzjs/geojson": "^4.16.0",
+    "node-fetch": "^2.6.1",
```
Comment (blacha) [inline packages/lambda-tiler/package.json:25]: This should be a devDependency as it is not needed at runtime.

Comment (Wentao-Kuang) [reply packages/lambda-tiler/package.json:25]: Good pick.

Comment (jacott) [reply packages/lambda-tiler/package.json:25]: But isn't it needed at runtime because the test is run at runtime?

**Review by jacott — COMMENTED** [review]
What about a test for `api.ts`?

**Pull request closed without merging**